### DISCUSSION TITLE
Image upload error

### DISCUSSION THREAD

**Discussion by RajGM** [AntlerVC/firetable#965] [Support / Q&A] [answered]
Hi @notsidney 

An unknown error occurred when uploading an image using the image field.
And the same error occurs when uploading a file using the File field.

I have enabled the Google Cloud Run services with cloud storage as well.

**Comment by BeeBombshell** [comment]
Hey @RajGM, what error are you getting? Can you provide some screenshots or perhaps a video recording?

**Reply by RajGM** [reply]
> Hey @RajGM, what error are you getting? Can you provide some screenshots or perhaps a video recording?

It was https://github.com/rowyio/rowy/discussions/965#discussioncomment-4195063
Then I checked consolelog
Found that Google Cloud Storage was not initialized
Then Image started uploading, but Rowy was not able to access
Then I changed the rules, and it worked.

**Comment by RajGM** [comment, marked as answer]
But still, I am getting errors on the console.

`
{
  "error": {
    "code": 403,
    "message": "Permission denied."
  }
}`

**Reply by BeeBombshell** [reply]
Hey @RajGM, is this still an issue?

**Reply by RajGM** [reply]
Hi @BeeBombshell 
It's been solved.
